stock_orderpoint_manual_procurement/models/stock_warehouse_orderpoint.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+from datetime import datetime
+from odoo.addons import decimal_precision as dp
+from odoo.tools import float_compare, float_round
+
+
+UNIT = dp.get_precision('Product Unit of Measure')
+
+
+class StockWarehouseOrderpoint(models.Model):
+    _inherit = 'stock.warehouse.orderpoint'
+
+    @api.multi
+    @api.depends("product_min_qty", "product_id", "qty_multiple")
+    def _compute_procure_recommended(self):
+        op_qtys = self.subtract_procurements_from_orderpoints()
+        for op in self:
+            op.procure_recommended_date = op._get_date_planned(
+                datetime.today())
+            procure_recommended_qty = 0.0
+            prods = op.with_context(
+                location=op.location_id.id).product_id.virtual_available
+            if prods is None:
```

Comment:
@lreficent As product_id is required on orderpoints and virtual_available is at least == 0.0, this is not necessary

Reply:
you're right